20_Simulation_Principles/50_Multi-Model_Representation/10_Physics_integration.md

```diff
@@ -5,98 +5,86 @@ Conservation of linear momentum
+$$\frac{d\boldsymbol{p}}{dt}=f_{\text{vol}}+f_{\text{surf}}$$
+$$\frac{D}{Dt}\int_{\Omega}\rho%20\boldsymbol{v}d\Omega%20=\int_{\Omega}%20\rho%20\boldsymbol{b}d\Omega%20+\int_{\Gamma}\boldsymbol{t}d\Gamma$$
+$$\int_{\Omega}%20\rho%20\frac{D\boldsymbol{v}}{Dt}d\Omega%20=\int_{\Omega}%20\rho%20\boldsymbol{b}d\Omega%20+\int_{\Gamma}n\cdot%20\boldsymbol{\sigma}d\Gamma$$
+$$\int_{\Omega}\rho%20\frac{D\boldsymbol{v}}{Dt}d\Omega%20=\int_{\Omega}\rho%20\boldsymbol{b}d\Omega%20+\int_{\Omega}\frac{\partial%20\sigma_{ij}}{\partial%20x{i}}d\Omega$$
+$$\rho%20\dot{v}=\rho%20\boldsymbol{b}+\nabla%20\cdot%20\boldsymbol{\sigma}$$
 
-<img class="latex" src="https://latex.codecogs.com/png.latex?\frac{d\boldsymbol{p}}{dt}=f_{\text{vol}}+f_{\text{surf}}" title="Newton's second law" />
 
-<img class="latex" src="https://latex.codecogs.com/png.latex?\frac{D}{Dt}\int_{\Omega}\rho%20\boldsymbol{v}d\Omega%20=\int_{\Omega}%20\rho%20\boldsymbol{b}d\Omega%20+\int_{\Gamma}\boldsymbol{t}d\Gamma" title="Integral form" />
 
-<img class="latex" src="https://latex.codecogs.com/png.latex?\int_{\Omega}%20\rho%20\frac{D\boldsymbol{v}}{Dt}d\Omega%20=\int_{\Omega}%20\rho%20\boldsymbol{b}d\Omega%20+\int_{\Gamma}n\cdot%20\boldsymbol{\sigma}d\Gamma" title="Cauchy's law" />
+where $$\boldsymbol{\sigma}$$ is the Cauchy's stress tensor (as a reminder $$\nabla%20\cdot%20\boldsymbol{\sigma}=\text{div}(\boldsymbol{\sigma})$$), $$\boldsymbol{b}$$ is the density of external forces (force per mass unit), $$\rho$$ is the mass density and $$\dot{v}$$ is the acceleration of the body. The conservation of linear momentum equation relates the change of momentum (LHS inertia term) and the equilibrium equation (RHS with the internal and external forces), which must be solved at each time step.
 
-<img class="latex" src="https://latex.codecogs.com/png.latex?\int_{\Omega}\rho%20\frac{D\boldsymbol{v}}{Dt}d\Omega%20=\int_{\Omega}\rho%20\boldsymbol{b}d\Omega%20+\int_{\Omega}\frac{\partial%20\sigma_{ij}}{\partial%20x{i}}d\Omega" title="Gauss's theorem" />
 
-<img class="latex" src="https://latex.codecogs.com/png.latex?\rho%20\dot{v}=\rho%20\boldsymbol{b}+\nabla%20\cdot%20\boldsymbol{\sigma}" title="Strong form" />
 
+In order to find a solution over the domain of simulation, we need to integrate this momentum equation. However, in much cases, no exact solution can directly be found. To ensure the existence of a solution, this strong form must be converted into a discrete problem, known as the weak or variational formulation. This is done by taking the product of a test function with the momentum equation and integrating over the current configuration. It is equivalent to formulating the problem to require a solution in the sense of a distribution. The test function is also known as weight function or basis functions, noted $$\psi_j$$. Let $$\Omega$$ denote the volume of our domain and $\Gamma$ denote the surface of the domain, the weak form of the momentum equation becomes:
```

Comment:
"it is quivalent to formulation the problem to require a solution in the sense of a distribution" : I am sorry, but after reading it multiple times, I am still not sure of what it means. Either remove this or reformulate. 

Typo in the next sentence : function is singular and then plural.

Reply:
The diff is misleading, I though it was added by this PR... But it is a cleaning PR so, let's clean :D